pelux.xml

```diff
@@ -31,10 +31,10 @@
            path="sources/meta-bistro"/>
```

Comment:
meta-bistro should also be moved to thud.

Reply:
yes, I don't wanna branch them atm; I had to branch `meta-pelux` to remove `mopidy`.

Reply:
Just to clarify we won't make a branch on an internal layer just to have a branch with a certain name. While there isn't a release to be done soon we will point to master and before the next release we will create the thud branch, but since there are no changes needed to adapt to thud then there is no point in making the branch at this time.